files/chef-marketplace-cookbooks/chef-marketplace/recipes/config.rb

```diff
@@ -0,0 +1,12 @@
+directory '/etc/chef-marketplace' do
+  owner 'root'
+  group 'root'
+  mode '0775'
+  action :create
+end
+
+if File.exist?('/etc/chef-marketplace/marketplace.rb')
+  Marketplace.from_file('/etc/chef-marketplace/marketplace.rb')
```

Comment:
Nitpick: I would move the config to a submodule (i.e. Marketplace::Config) so it's clear what from_file is for.